src/components/store/AddCartButton/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { Dropdown } from '@/components/common';
+import { useEffect, useId, useState } from 'react';
+import styles from './style.module.scss';
+
+interface AddCartButtonProps {
+  currSize: string | undefined;
+  // Justification for disabling rules: This seems to be a false positive.
+  // https://stackoverflow.com/q/63767199/
+  // eslint-disable-next-line no-unused-vars
+  inStock: boolean;
+  inCart: boolean;
+  lifetimeRemaining: number;
+  montlyRemaining: number;
+  setInCart: (value: boolean) => void;
+}
+
+const AddCartButton = ({
+  currSize,
+  inStock,
+  inCart,
+  setInCart,
+  lifetimeRemaining,
+  montlyRemaining,
+}: AddCartButtonProps) => {
+  const [mounted, setMounted] = useState(false);
+  const [amount, setAmount] = useState<number>(1);
+
+  const myID = useId();
+
+  const maxCanBuy = Math.min(lifetimeRemaining, montlyRemaining);
+
+  useEffect(() => {
+    // console.log(`currSize ${currSize} inStock ${inStock} inCart ${inCart}`);
```

Comment:
delete unused comments

a tip for future PRs you open - try to do a full self review before you have anyone else look at it and if you get a comment, see everywhere it applies and fix it so you don't need to read the same comment written multiple times

Reply:
If someone could show me how to get to the view y'all are using to make small commits, that would be helpful for this.

Reply:
not really a special view or anything, but reviewing all the changes you made https://github.com/acmucsd/membership-portal-ui-v2/pull/94/files can be helpful (albeit it can be a lot to look over), because these are the changes we are reviewing, and it highlights the lines you made. so you could use this to find all the comments you added, and see if they're unused or not

for seeing if a suggestion can be applied elsewhere in your code, you can go back into VS Code and search across all the files (ctrl+shift+F). for example, you could use this to look up some kinds of comments, like `// Justification for disabling rules`, to see if they're necessary or not

for the little suggested commits we make in reviews, we do that by doing

````md
```suggestion
suggested change here
```
````

and then github will show a button to make the commit